Updated the Dutch translations

Rubenoo opened the pull request:
Small changes to the dutch translations, made with love <3

PS sorry for the 2 commits i hope that its not a problem

fat-tire commented:
Hey rubenoo--  commit #1 and commit #2 seem to both edit the dutch translation file?  Are the changes identical?  Could you possibly separate them out, for clarity?  Not to make you do a lot of work here... if you want I could make one commit for that file.

Rubenoo commented:
The first commit (Merge pull request #1 from cSploit/develop) was to update my own repo to the latest version of the Csploit (developer). After that i re-checked the dutch translations and fixed a bad xml bug and corrected the translations(commit 2) The changes that are made in this pull request are correct

tux-mind commented:
thank you very much @Rubenoo :blush:
